x-pack/plugins/search_inference_endpoints/public/components/all_inference_endpoints/tabular_page.tsx

```diff
@@ -46,12 +98,32 @@ export const TabularPage: React.FC<TabularPageProps> = ({ inferenceEndpoints })
   );
 
   return (
-    <EndpointsTable
-      columns={TABLE_COLUMNS}
-      data={paginatedSortedTableData}
-      onChange={handleTableChange}
-      pagination={pagination}
-      sorting={sorting}
-    />
+    <EuiFlexGroup direction="column">
+      <EuiFlexItem>
+        <EuiFlexGroup gutterSize="s">
+          <EuiFlexItem grow={false}>
```

Comment:
Let's add a `style={{ width: "400px" }}` here.  The underlying search input will fill the space and so it's recommended to define the widths on the flex items. 

This means we should also be able to delete that scss file with single width defined.

Reply:
Updated

Reply:
Ideally we don't do hardcoded pixel values for width but rely on calcullations based on EUI values.